pkg/util/stdio/arm_nondarwin.go

```diff
@@ -0,0 +1,11 @@
+// +build linux,arm64 windows
```

Comment:
Is there any use for Windows on arm? Never heard of it to be honest.

Reply:
There's an emulation layer for devices running windows on arm64 (such as `surface pro x`) to run x64 binaries, but the latest windows (10 & 11) does support devices on arm64

Reply:
@alexadhy does not hurt to support it if it is an option. Lets keep it.